VMware docs update changes (#4256)
SumoLogic/sumologic-documentation: blog-service/2024-07-01-apps.md

---

> title: VMware (Apps)

Reviewer: ```suggestion
title: VMware vSphere version 8 (Apps)
```

image: https://help.sumologic.com/img/sumo-square.png
keywords:
  - apps

>   - app catalog

Reviewer: ```suggestion
  - app-catalog
```

  - vmware
hide_table_of_contents: true
authors:
  - url: https://help.sumologic.com/release-notes-service/rss.xml
    image_url: /img/release-notes/rss-orange.png
---

> We are excited to announce that our VMware app now fully supports VMware vSphere version 8!. We have fixed CVEs and updated the libraries used for collecting data from VMware, ensuring a more secure and efficient virtualization environment.

Reviewer: ```suggestion
We are excited to announce that our VMware app fully supports VMware vSphere version 8. We have fixed CVEs and updated the libraries for collecting data from VMware, ensuring a more secure and efficient virtualization environment. [Lear more](/docs/integrations/containers-orchestration/vmware).
```
